src/batch-client/index.ts

```diff
@@ -141,8 +142,10 @@ export class ZimbraBatchClient {
 	private dataLoader: DataLoader<RequestOptions, RequestBody>;
 	private jwtToken?: string;
 	private notificationHandler?: NotificationHandler;
+	private serverVersion?: string;
 
 	constructor(options: ZimbraClientOptions = {}) {
+		parseUserAgent();
```

Comment:
api-client should be not do any browser parsing and add data related to that in requests, client should parse it and pass details to api-client

Reply:
will update